### ISSUE TITLE
Cannot compile hello_world on mac

### ISSUE THREAD

**Issue by Nathan22Miles** [google/mediapipe#1136]
I am running macOS Mojave 10.14.6

As was able to follow the install instructions but when I try to compile hello_world I see the following.
It looks like it thinks I want to run on Android, when I actually want to run on the desktop.

mediapipe commit: 8f69af91feb5944e130959dc9cc256b93d03abd2

bazel 3.5.0-homebrew

```
➜  mediapipe git:(master) bazel run --define MEDIAPIPE_DISABLE_GPU=1 mediapipe/examples/desktop/hello_world:hello_world 
Starting local Bazel server and connecting to it...
INFO: SHA256 (https://github.com/bazelbuild/rules_foreign_cc/archive/master.zip) = f358144776d3dfc5a928ef32a0e4fbe93c99b55772d70cca77a6478e34d96aa7
DEBUG: Rule 'rules_foreign_cc' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "f358144776d3dfc5a928ef32a0e4fbe93c99b55772d70cca77a6478e34d96aa7"
DEBUG: Repository rules_foreign_cc instantiated at:
  no stack (--record_rule_instantiation_callstack not enabled)
Repository rule http_archive defined at:
  /private/var/tmp/_bazel_nmiles/44f14cdbe8ed532118a38417b5444a80/external/bazel_tools/tools/build_defs/repo/http.bzl:336:31: in <toplevel>
INFO: SHA256 (https://github.com/bazelbuild/rules_cc/archive/master.zip) = 97ddba49baf32121709b8578db6b6a58a4b9ff192de14199e6c72b9c7f221c99
DEBUG: Rule 'rules_cc' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "97ddba49baf32121709b8578db6b6a58a4b9ff192de14199e6c72b9c7f221c99"
DEBUG: Repository rules_cc instantiated at:
  no stack (--record_rule_instantiation_callstack not enabled)
Repository rule http_archive defined at:
  /private/var/tmp/_bazel_nmiles/44f14cdbe8ed532118a38417b5444a80/external/bazel_tools/tools/build_defs/repo/http.bzl:336:31: in <toplevel>
ERROR: Analysis of target '//mediapipe/examples/desktop/hello_world:hello_world' failed; build aborted: Either the path attribute of android_sdk_repository or the ANDROID_HOME environment variable must be set.
INFO: Elapsed time: 15.415s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (8 packages loaded, 11 targets configured)
FAILED: Build did NOT complete successfully (8 packages loaded, 11 targets configured)
    currently loading: @bazel_tools//tools/python ... (3 packages)
    Fetching @local_config_cc_toolchains; Restarting.
    Fetching @local_execution_config_python; fetching
```

**Comment by jiuqiant** [comment]
It's a Bazel 3.5.0 issue and will be solved in Bazel 3.6.0 (or you can go back to Bazel 3.4.0 if Bazel 3.6.0 is still not available). See https://github.com/google/mediapipe/issues/1094#issuecomment-690480208

**Comment by Nathan22Miles** [comment]
That fixed it. Thanks! This is an amazing piece of technology.
